ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -76,12 +76,17 @@
     private String features;
     private CCDYesNoOption migratedFromClaimStore;
     private List<CCDCollectionElement<CCDClaimDocument>> caseDocuments;
-    private List<CCDCollectionElement<CCDScannedDocument>> scannedDocuments;
+    @Builder.Default
+    private List<CCDCollectionElement<CCDScannedDocument>> scannedDocuments = Collections.emptyList();
+    @Builder.Default
+    private List<CCDCollectionElement<CCDScannedDocument>> filteredScannedDocuments = Collections.emptyList();
```

Comment:
Maybe rename _filteredScannedDocuments_ to _temporaryFilteredScannedDocuments_ since it seems that this property is only used during this event.